app/src/main/java/com/bunbeauty/papakarlo/feature/consumer_cart/ui/CartProductItem.kt

```diff
@@ -37,17 +33,12 @@ fun CartProductItem(
     cartProductItem: CartProductItem,
     onCountIncreased: () -> Unit,
     onCountDecreased: () -> Unit,
-    onClick: () -> Unit
+    onClick: () -> Unit,
 ) {
-    Card(
+    FoodDeliveryCard(
         modifier = modifier
-            .card()
-            .clickable(
-                interactionSource = remember { MutableInteractionSource() },
-                indication = rememberRipple(),
-                onClick = onClick
-            ),
-        colors = FoodDeliveryTheme.colors.cardColors()
+            .card(),
```

Comment:
It's not a good idea to use this extension. 
1) It's not clear what this function do and you need check its implementation every time. 
2) We save only 1 line of code

Reply:
+